### PR TITLE
Add integration tests for `ContractFunctionParameters`

### LINKED ISSUE #1477: Add integration tests for `ContractFunctionParameters`
### Problem

`ContractFunctionParameters` is missing integration tests.

### Solution

To add integration tests for `ContractFunctionParameters`.

For reference see [JS SDK](https://github.com/hashgraph/hedera-sdk-js/blob/75fcb17125e3f25bb2642f4bd3ba64bd94ce2b93/test/integration/ContractFunctionParametersIntegrationTest.js) and [Go SDK](https://github.com/hashgraph/hedera-sdk-go/blob/550679fe4b0b0eab874000d7ac11ea93af45ed14/contract_function_parameters_e2e_test.go).

### Alternatives

N/A

### PR DESCRIPTION
**Pull request by thenswan**
**Description**:
- add `getResult` function for `ContractFunctionParameters`;
- add integration tests for `ContractFunctionParameters`;

**Related issue(s)**:
Fixes #1477

**Notes for reviewer**:
Will provide in comments.

**Checklist**
- [ ] Documented (Code comments, README, etc.)
- [X] Tested (unit, integration, etc.)

### CODE REVIEW

File: sdk/src/integrationTest/java/ContractFunctionParametersIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,2233 @@
...
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+// Check Go & JS examples for workaround re not implemented funcs
+// Test int and uint arrays?
+// Revisit test func names
+public class ContractFunctionParametersIntegrationTest {
```
[the remark this answers is not in the document]
Comment (thenswan) [reply sdk/src/integrationTest/java/ContractFunctionParametersIntegrationTest.java:25]: `GetResult` method should be added (like in other JS or Go SDKs), waiting for @ochikov to get response re best suiting dependency.

[the remark this answers is not in the document]
Comment (thenswan) [reply sdk/src/integrationTest/java/ContractFunctionParametersIntegrationTest.java:25]: I think revisiting test function names and descriptions is a good idea.

File: sdk/src/integrationTest/java/ContractFunctionParametersIntegrationTest.java [hunk 2]
```diff
@@ -0,0 +1,2233 @@
...
+            .setQueryPayment(new Hbar(10))
+            .execute(testEnv.client);
+
+        assertThat(Integer.toUnsignedString(response.getUint32(0))).isEqualTo(uint32Max);
+        assertThat(response.getUint64(1)).isEqualTo(Long.parseUnsignedLong(uint32Max) - 1);
+        assertThat(response.getString(2)).isEqualTo("OK");
+    }
+
+    /*
+    ContractFunctionParameters#AddInt256BigInt && #AddInt256BigUInt is not implemented (re tests for BigInt256, BigUint256 and MultipleBigInt256)
```
[the remark this answers is not in the document]
Comment (thenswan) [reply sdk/src/integrationTest/java/ContractFunctionParametersIntegrationTest.java:2082]: Can ignore it and remove comment.

**Review by thenswan — COMMENTED** [review]
WIP after sync w/ @bamzedev and @ochikov

**Comment by thenswan** [comment]
Revisit function names and test description copy or all good?

**Review by petreze — APPROVED** [review]

File: sdk/src/integrationTest/java/ContractFunctionParametersIntegrationTest.java [hunk 3]
```diff
@@ -0,0 +1,2774 @@
...
+
+        var response = new ContractCallQuery().setContractId(contractId).setGas(1500000)
+            .setFunction("returnBytes32Arr", new ContractFunctionParameters().addBytes32Array(testBytesLen32))
+            .setQueryPayment(new Hbar(10)).execute(testEnv.client);
+
+        var responseResult = (byte[][]) response.getResult("(bytes32[])").get(0);
+
+        assertThat(responseResult).isEqualTo(testBytesLen32);
+    }
+}
```
Comment (bamzedev) [inline sdk/src/integrationTest/java/ContractFunctionParametersIntegrationTest.java:2774]: I would add strings and boolean

Comment (thenswan) [reply sdk/src/integrationTest/java/ContractFunctionParametersIntegrationTest.java:2774]: (for `getResult` function)

Comment (thenswan) [reply sdk/src/integrationTest/java/ContractFunctionParametersIntegrationTest.java:2787]: @bamzedev added for strings, but noticed that Java SDK is lacking `addBoolArray` function, so test for boolean array wasn't added.

cc @rokn

**Pull request merged**